source/transfer_list.rst

```diff
@@ -732,15 +732,15 @@ This address is used when OPTEE represents SPMC at S-EL1 and SPMD is the Secure
    * - tos_fw_config
      - data_size
      - hdr_size
-     - TOS FW config address for SPMD
+     - TOS FW config base address
 
 .. _opteed_aarch_entry:
 
 OPTEED AARCH entry layout (XFERLIST_OPTEED_AARCH)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 This entry type holds the AARCH mode for OPTEED.
-This AARCH mode is used when OPTEED is the Secure Payload Dispatcher.
```

Comment:
What is OPTEED?

In general, if you an acronym, please can you write it out in full at least once?

Reply:
Added the definitions.